Resources/Prototypes/_Goobstation/Entities/Structures/Machines/material_silo.yml

```
@@ -47,6 +50,7 @@
     range: 1000
   - type: DeviceLinkSink
     ports:
+      - FillbotAnyItem
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)